### PR TITLE
Make error class of div in FormHelper::input configurable

### PR DESCRIPTION
**Pull request by mvdriel**
Make error class of div in FormHelper::input configurable, so it is possible to generate html snippets compatible with Bootstrap 4.

### CODE REVIEW

File: lib/Cake/View/Helper/FormHelper.php [hunk 1]
```diff
@@ -1089,6 +1089,7 @@ public function input($fieldName, $options = array()) {
 		if (!empty($divOptions['tag'])) {
 			$tag = $divOptions['tag'];
 			unset($divOptions['tag']);
+			unset($divOptions['errorClass']);
```
Comment (markstory) [inline lib/Cake/View/Helper/FormHelper.php]: This can be combined with the `unset()` 1 line above.

**Comment by markstory** [comment]
Related to #2031

File: lib/Cake/View/Helper/FormHelper.php (hunk 1 above)
Comment (mvdriel) [inline lib/Cake/View/Helper/FormHelper.php]: Yes, thanks, done.

**Comment by tersmitten** [comment]
Thanks @markstory

**Pull request merged**